umrr_ros2_driver/src/smartmicro_radar_node.cpp

```diff
@@ -166,6 +171,126 @@ SmartmicroRadarNode::SmartmicroRadarNode(const rclcpp::NodeOptions & node_option
     m_publishers[i] = create_publisher<sensor_msgs::msg::PointCloud2>(
       "umrr/targets_" + std::to_string(i), sensor.history_size);
   }
+
+  // create a ros2 service to change the radar parameters
+  command_srv_ = create_service<umrr_ros2_msgs::srv::SetParam>(
+    "smartmicro_radar_node/set_radar_parameter",
+    std::bind(
+      &SmartmicroRadarNode::change_radar_mode, this, std::placeholders::_1,
+      std::placeholders::_2));
+  RCLCPP_INFO(rclcpp::get_logger("rclcpp"), "Radar parameter services are ready.");
```

Comment:
Because this is part of `SmartMicroRadarNode` which inherits from `rclcpp::Node`, you should use `this->get_logger()` here instead because it comes populated with the correct logger name so the log output starts with the node name.